src/typeorm/typeorm.config.ts

```diff
@@ -1,5 +1,9 @@
 import { ConfigModule, ConfigService } from '@nestjs/config';
 import { TypeOrmModuleAsyncOptions, TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { DataSourceOptions } from 'typeorm';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
Seems to have repeated. Wonder if this can be used once somewhere

Reply:
You are right! Hmmm. I'll look into it and change it in the next PR.